test/mailers/notifier_mailer_test.rb

```diff
@@ -138,14 +138,27 @@ class NotifierMailerTest < ActionMailer::TestCase
     assert response.subject.include?(
       I18n.t(
         'notifier.notify_new_finding_answer.title',
-        :review => finding_answer.finding.review.to_s
+        review: finding_answer.finding.review,
+        finding_id: finding_answer.finding.id
       )
     )
     assert_match Regexp.new(I18n.t('notifier.notify_new_finding_answer.finding_link')),
       response.body.decoded
     assert_equal user.email, response.to.first
   end
 
+  test 'notify action not found' do
+    user    = users(:administrator)
+    comment = 'comment'
+
+    response = NotifierMailer.notify_action_not_found(user.email, comment).deliver_now
+
+    assert !ActionMailer::Base.deliveries.empty?
```

Comment:
```suggestion
    refute ActionMailer::Base.deliveries.empty?
```

Aunque los otros estén así, vamos con este de ahora en adelante.

Reply:
impecable, modifico solo de los que hice.